hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSPermissionChecker.java

```diff
@@ -89,6 +89,7 @@ private String toAccessControlString(INodeAttributes inodeAttrib,
   private final Collection<String> groups;
   private final boolean isSuper;
   private final INodeAttributeProvider attributeProvider;
+  private final AccessControlEnforcer  accessControlEnforcer;
```

Comment:
How about using a space between `AccessControlEnforcer` and `accessControlEnforcer`?

Reply:
@jojochuang, @ZanderXu  - thank you for reviewing the PR.

Updated to use a single space between `AccessControlEnforcer` and `accessControlEnforcer`.